pulsar-zookeeper-utils/src/main/java/com/yahoo/pulsar/zookeeper/ZooKeeperCache.java

```diff
@@ -289,28 +290,38 @@ public Boolean call() throws Exception {
 
             // Broker doesn't restart on global-zk session lost: so handling unexpected exception
             try {
-                this.zkSession.get().getData(path, watcher, (rc, path1, ctx, content, stat) -> {
+                // Put watch on the node no matter what
```

Comment:
Can you describe this change a bit more? Since it doesn't affect only the bundle splitting in the load manager, it would be better to split it in a separate PR.